### ISSUE TITLE
Managed Media-Elemente ebenfalls thumbhashen

### ISSUE THREAD

**Issue by AWqxKAWERbXo** [FriendsOfREDAXO/thumbhash#2]
Moin @aeberhard, ich hatte mich selbst an einer Integration in mein Addon `media_manager_responsive` bemüht und bin da ein Jahr nicht vorangekommen. https://github.com/alexplusde/media_manager_responsive/pull/29

Jetzt habe ich überlegt, einfach thumbhash optional über dein Addon beizufügen.

Sehe ich das richtig, dass dein Addon momentan nur die Ursprungsmedien berücksichtigt, aber keine Managed-Media-Objekte? 

Wäre es dir möglich, thumbhash um Managed-Media-Objekte zu erweitern, bspw. indem es im passenden Medienprofil ein `mediafile.ext.thumbhash` zur Laufzeit generiert wird, das den data-img-Code enthält?

**Comment by aeberhard** [comment]
Hi @alxndr-w.
Das Addon erweitert den Medienpool um 2 Felder in dem die notwendigen Werte gespeichert werden.
Bei Upload oder Änderung eines Bildes werden die Daten generiert und gespeichert, wegen der Performance.
Werden Managed-Media Objekte auch in einer Tabelle gespeichert? Dann kann man das sicher erweitern.
Bin noch bis 16.08. in Urlaub ...

**Comment by AWqxKAWERbXo** [comment]
Nein, die werden ausschließlich auf dem Dateisystem gespeichert. 

Dann genieße erstmal deinen Urlaub. 👍🏻
